scripts/searchindex.ts

```diff
@@ -245,6 +247,34 @@ function getCurrentDate(): string {
     });
   });
 
+  allSolutions.forEach(async (solution) => {
+    const body = solution.body.raw.replace(/<\/?[^>]+(>|$)/g, '');
+    const process = execa('pandoc', ['-f', 'markdown', '-t', 'plain', '--wrap=none']);
+    process.stdin.write(body);
+    process.stdin.end();
+    const result = await process;
+
+    const url = new URL(solution.url);
+    await client.index({
+      index,
```

Comment:
If we're putting this into the existing `site-search-dev-blog-*` index, we probably want to consider renaming it if it contains for than blog content.